packages/components/src/tooltip/README.md

```diff
@@ -46,6 +46,14 @@ The tooltip text to show on focus or hover.
 -   Type: `String`
 -   Required: No
 
+### delay
```

Comment:
We should likely either (A) implement this property for native or (B) add a note here that `delays` is a web-only property ([similar to `visible`](https://github.com/WordPress/gutenberg/blob/22774dc79ae9c2d65741de8e903d8bc7c1e774a7/packages/components/src/tooltip/README.md#visible-native-only)). 

I would prefer the former to keep the platforms aligned on API as much as possible, and I believe it would be fairly straightforward to accomplish. In the sibling `index.native.js` file variant, we would likely add the same `delay` property with a default value of a new `TOOLTIP_DELAY` constant and reference it on the line below instead of `500`. 

https://github.com/WordPress/gutenberg/blob/3ac66d9f0a8f1f8834ceffa48a901f2d95ccb267/packages/components/src/tooltip/index.native.js#L159

WDYT? Would you be willing to implement this property for native? If so, the [getting started with native documentation](https://github.com/WordPress/gutenberg/blob/22774dc79ae9c2d65741de8e903d8bc7c1e774a7/docs/contributors/code/getting-started-native-mobile.md) may be helpful.

Reply:
Good catch. I have not worked on React Native before. So it will be a challenge for me. What I might do here is,

- Add the web-only note here for `delay` prop for now, so that this PR is unblocked.
- Will start digging into the React Native resource you have shown and will try if I can get a PR up for native as well.